test/lib/commands/config.js

```diff
@@ -394,10 +390,6 @@ t.test('config edit', async t => {
 })
 
 t.test('config edit - editor exits non-0', async t => {
-  t.teardown(() => {
-    spawk.clean()
-  })
-
```

Comment:
Previously spawk was calling clean after the teardown of some invididual tests. Is that the same thing as calling `spawk.done()` in `afterEach` and then `spawk.unload()` once at the end of all the tests?